packages/vitest/src/index.mjs

```diff
@@ -0,0 +1,73 @@
+import vitest from '@vitest/eslint-plugin';
+
+/**
+ * @type {import('eslint').Linter.Config[]}
+ */
+const config = [
+  vitest.configs.recommended,
+
+  {
+    name: '@metamask/eslint-config-vitest',
+
+    files: [
+      '**/*.test.js',
+      '**/*.spec.js',
+      '**/*.test.mjs',
+      '**/*.spec.mjs',
+      '**/*.test.cjs',
+      '**/*.spec.cjs',
+      '**/*.test.ts',
+      '**/*.spec.ts',
+      '**/*.test.tsx',
+      '**/*.spec.tsx',
+      '**/*.test.mts',
+      '**/*.spec.mts',
+      '**/*.test.cts',
+      '**/*.spec.cts',
+      '**/*.test.mtsx',
+      '**/*.spec.mtsx',
+      '**/*.test.ctsx',
+      '**/*.spec.ctsx',
+    ],
+
+    rules: {
+      'vitest/consistent-test-it': ['error', { fn: 'it' }],
+      'vitest/no-alias-methods': 'error',
+      'vitest/no-commented-out-tests': 'error',
+      'vitest/no-conditional-expect': 'error',
+      'vitest/no-conditional-in-test': 'error',
```

Comment:
```suggestion
      'vitest/no-conditional-in-test': 'error', // Corresponding to "jest/no-if"
```

Reply:
`jest/no-if` is replaced with `jest/no-conditional-in-test` in the latest version.